src/ledger/LedgerTxnTrustLineSQL.cpp

```diff
@@ -54,7 +58,8 @@ LedgerTxnRoot::Impl::loadTrustLine(LedgerKey const& key) const
     ZoneScoped;
     std::string accountIDStr, issuerStr, assetStr;
     getTrustLineStrings(key.trustLine().accountID, key.trustLine().asset,
-                        accountIDStr, issuerStr, assetStr);
+                        accountIDStr, issuerStr, assetStr,
+                        mHeader->ledgerVersion);
```

Comment:
I'm pretty sure this is alright but wanted to point it out - mHeader is never null here right?

Reply:
Yes, this is fine. `mHeader` can never be null, under any circumstances.